radis/lbl/broadening.py

```diff
@@ -2390,7 +2396,9 @@ def calculate_pseudo_continuum(self, noneq=False):
 
         if self.params.pseudo_continuum_threshold > 0:
 
-            self.profiler._print(2, "Calculating pseudo continuum")
+            if self.verbose >= 2:
+                print("... Calculating pseudo continuum")
+
```

Comment:
Hello. Can you add an optional "start" message to `self.profiler.start` instead ?  It would remove all these lines and make the source code cleaner. Also, can you remove t0 = time() as it should be embedded in the Profiler too ?

Reply:
Did you mean `self.profiler.start("key",vebose_level, optional="")`. We can use it to print the statements prior to `start()` but then we need to make sure the verbose level matches of start and the prior statement and if it matches then we can simply print it using `("... Calculating pseudo continuum")` or `"..." * (verbose_level -1) + optional` which will be similar to `_print` method. So should not use the previous proposed [implementation](https://github.com/radis/radis/pull/286#issuecomment-868400588), will cover a larger area? Or the above discussed `optional` method?